### PR TITLE
testcase/network: fix svace issue

### PR DESCRIPTION
**Pull request by JeonginKim**
fix handle_leak issue in network TC

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
apps/examples/testcase/tc_common.h:45: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:86: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:98: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:110: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:122: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:134: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
apps/examples/testcase/tc_common.h:146: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'

File: apps/examples/testcase/le_tc/network/tc_net_recvfrom.c [hunk 1]
```diff
@@ -125,6 +126,8 @@ void *recvfrom_udpserver(void *args)
 	tc_net_recvfrom_p(SocketFD);
 	tc_net_recvfrom_sock_n();
 	tc_net_recvfrom_n(SocketFD);
+	TC_ASSERT_EQ_CLEANUP_RETURN("socket", SocketFD, -1, close(SocketFD), 0);
```
Comment (pillip8282) [inline apps/examples/testcase/le_tc/network/tc_net_recvfrom.c]: you don't need to check the result of socket creation.
I think that just closing the socket is enough

File: apps/examples/testcase/le_tc/network/tc_net_recvfrom.c [hunk 2]
```diff
@@ -154,6 +157,8 @@ void *recvfrom_udpclient(void *args)
 	dest.sin_port = htons(PORTNUM);
 	fromlen = sizeof(dest);
 	sendto(mysocket, buffer, len, 0, (struct sockaddr *)&dest, fromlen);
+	TC_ASSERT_EQ_CLEANUP_RETURN("socket", mysocket, -1, close(mysocket), 0);
```
Comment (pillip8282) [inline apps/examples/testcase/le_tc/network/tc_net_recvfrom.c]: same as above

File: apps/examples/testcase/tc_common.h [hunk 3]
```diff
@@ -54,6 +54,16 @@ extern int total_fail;
 	} \
 }
 
+#define TC_ASSERT_EQ_CLEANUP_RETURN(api_name, var, ref, freeResource, ret) \
```
Comment (pillip8282) [inline apps/examples/testcase/tc_common.h]: we don't need this if you delete above codes

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**